packages/cel/src/list.ts

```diff
@@ -38,6 +39,14 @@ export interface CelList extends Iterable<CelValue> {
    * is out of range.
    */
   get(index: number): CelValue | undefined;
+  /**
+   * Determines if a value is in a list.
+   */
+  has(value: CelValue): boolean;
+  /**
+   * Determines if a value in the list yields true for a callback.
+   */
+  some(callback: (v: CelValue) => boolean): boolean;
```

Comment:
I don't think we should add these here. These are slowly being added to iterators and in fact `some` is already part of 2025 baseline: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Iterator/some. One other reason is that we will have to justify why other such methods are not supported. I want to keep this as minimal as possible.

That said, we can always create an internal functions for this.